.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+    pull_request:
+    push:
+
+jobs:
+    test:
+        name: Run Tests
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v3
+
+            - name: Setup Node.js
+              uses: actions/setup-node@v3
+              with:
+                  node-version: $(cat .nvmrc)
```

Comment:
We should use `node-version-file` here: https://github.com/actions/setup-node?tab=readme-ov-file#usage